lib/relooper/src/graph/cfg/mod.rs

```diff
@@ -183,12 +191,18 @@ impl<T: Eq + Hash + Clone> Cfg<T> {
             .get(label)
             .expect("any node should have outgoing edges")
     }
+
+    pub fn edge_mut(&mut self, label: &T) -> &mut CfgEdge<T> {
+        self.out_edges
+            .get_mut(label)
+            .expect("any node should have outgoing edges")
```

Comment:
Every node should have outgoing edges, but not necessarily with any given label right? Seems like this function could panic with a misleading error message.

Reply:
you are right, but currently we only iterate over already present in cfg labels and im not sure if it will change in future (and it will blow up with such a message, there is probably a serious bug)
also, we can treat labels as nodes because we are enforcing that every label should have edge out of it (there was such a bug previously)
given all above, should we use only `labels` or only `nodes` to eliminate misunderstanding?

Reply:
Sure, it's probably fine to leave as is. Thanks for the clarifications!